scripts/AbinsModules/Instruments/Instrument.py

```diff
@@ -91,7 +91,8 @@ def _gaussian(self, sigma=None, points=None, center=None):
         :return: numpy array with calculated Gaussian values
         """
         sigma_factor = 2.0 * sigma * sigma
-        return 1.0 / np.sqrt(sigma_factor * np.pi) * np.exp(-(points - center) ** 2 / sigma_factor)
+        norm = points.size / (6 * sigma)
```

Comment:
@brandonhewer  here is a bug: points.size is:
points.size = number of frequencies * number of points divided per broadened peak

instead of  points.size it should be  a number of points divided per broadened peak, e.g,
AbinsModules.AbinsParameters.pkt_per_peak.

I will do some additional tests and commit a fix for that later today. Apart from that one line I also have to double check my modifications to scaling parameters for higher quantum order events, regenerate my reference data and check if they are OK.)